### PR TITLE
Support for CCX creation.

### PR DESCRIPTION
**Pull request by justinabrahms**
#### What are the relevant tickets?
https://github.com/mitodl/teachersportal/issues/457

#### What's this PR do?
Adds support for CCX Creation.
#### Where should the reviewer start?
ccx.py

#### How should this be manually tested?
Run it. It's majoritively copy-pasted from ccxcon.

#### Any background context you want to provide?
We'll use this in TP.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-  - TOX_ENV=py34
-  - TOX_ENV=py35
+matrix:
+  include:
+    - python: "2.7"
+      env: TOX_ENV=py27
+    - python: "3.4"
+      env: TOX_ENV=py34
+    - python: "3.5"
+      env: TOX_ENV=py35
```
Comment (justinabrahms) [inline .travis.yml]: python 3.5 isn't available unless you do this, which was causing a 'no-interpreter-found' error in other travis builds.

File: pylintrc [hunk 2]
```diff
@@ -1,4 +1,13 @@
-[BASIC]
+[MASTER]
+ignore=migrations,.git
```
Comment (giocalitri) [inline pylintrc]: I think we do not need `migrations`

**Comment by giocalitri** [comment]
no integration test?

**Comment by giocalitri** [comment]
code looks good

**Comment by justinabrahms** [comment]
Updated w/ integration test. Still need to figure out why course-detail fails.

**Comment by giocalitri** [comment]
I think this can be merged: the problem with course_detail seems unrelated :+1:

**Pull request merged**